test/spy-test.js

```diff
@@ -1650,6 +1650,114 @@ describe("spy", function () {
         });
     });
 
+    describe(".calledImmediatelyAfter", function () {
+        beforeEach(function () {
+            this.spy1 = createSpy();
+            this.spy2 = createSpy();
+            this.spy3 = createSpy();
+        });
+
+        it("is function", function () {
+            assert.isFunction(this.spy1.calledImmediatelyAfter);
+        });
+
+        it("returns true if first call to A was immediately after first to B", function () {
```

Comment:
This is minor, but it would make the tests easier to read, if you use `A` in both description and implementation. Otherwise readers would have to understand that `A` is `this.spy1`